packages/gestalt/src/FormHelperText.js

```diff
@@ -4,10 +4,16 @@ import { type Node } from 'react';
 import Box from './Box.js';
 import Text from './Text.js';
 
-export default function FormHelperText({ text }: {| text: string |}): Node {
+type Props = {|
+  text: string,
+  addA11yPause?: boolean,
+|};
+
+export default function FormHelperText({ text, addA11yPause = false }: Props): Node {
```

Comment:
For the future: maybe we should combine these two, to just FormHelperText, with either a default or error variant

Reply:
Since they seem so repetitive, and we only show one at a time